src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java

```diff
@@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
     @Test
     public void emptyLineSeparatorTest() throws IOException, Exception {
         
-        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
+        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
```

Comment:
Sounds like a bug in maven plugin, I do not see any reason it should be used in Check.